submissions/Eugene-CG/OOJS-Frogger-Game/app.js

```diff
@@ -0,0 +1,112 @@
+const charactersSize = {
+  charboy: 50,
+  bug: 35,
+  sizeDiff: 15,
+};
+const tileSize = {
+  width: 101,
+  height: 85,
+};
+const gameBoard = {
+  left: 0,
+  top: 0,
+  right: tileSize.width * 5,
+  bottom: tileSize.height * 6,
+};
+class Character {
+  constructor(x, y, sprite) {
+    this.x = x;
+    this.y = y;
+    this.sprite = sprite;
+  }
+  render() {
+    ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+  }
+  update() {}
+}
+class Enemy extends Character {
+  constructor(x, y, speed) {
+    super(x, y, "images/enemy-bug.png");
+    this.speed = speed;
+  }
+  collision() {
+    if (
+      this.x +
+        tileSize.width -
+        charactersSize.charboy +
+        charactersSize.sizeDiff >
+        player.x &&
+      this.x - charactersSize.charboy - charactersSize.sizeDiff < player.x &&
+      this.y === player.y + charactersSize.sizeDiff
+    ) {
+      setTimeout(() => {
+        player.x = tileSize.width * 2;
+        player.y = tileSize.height * 4 - charactersSize.charboy;
+      }, 100);
+    }
+  }
+  update(dt) {
+    this.x += this.speed * dt;
+    if (this.x > 500) this.x = -tileSize.width;
```

Comment:
What is the magic number 500?

Reply:
> What is the magic number 500?

Can I change code in github or better do it from vscode?
I wondering how to deal with this minor changes better

Reply:
It's up to you)